src/meta/RootUserMan.h

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef META_ROOTUSERMAN_H_
+#define META_ROOTUSERMAN_H_
+
+#include "base/Base.h"
+#include "kvstore/KVStore.h"
+#include "meta/MetaServiceUtils.h"
+#include "common/encryption/MD5Utils.h"
+
+namespace nebula {
+namespace meta {
+
+/**
+ * This class manages root user for meta server.
+ * */
+class RootUserMan {
+public:
+    static bool isUserExists(kvstore::KVStore* kv) {
+        auto userKey = MetaServiceUtils::userKey("root");
+        std::string val;
+        auto result = kv->get(kDefaultSpaceId, kDefaultPartId, userKey, &val);
+        if (result == kvstore::ResultCode::SUCCEEDED) {
+            LOG(INFO) << "Root user exists";
+            return true;
+        } else {
+            LOG(INFO) << "Root user is not exists";
```

Comment:
If current instance is not leader,  we should skip the check logic.

Reply:
> If current instance is not leader, we should skip the check logic.

Fixed. good catch. My lack of consideration.